CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+- Use a map instead of a proplist in the `room_event_data` hook
+  for the event data information.
+- `room_send_packet` hook removed.
+   Use `filter_room_packet` instead.
+
```

Comment:
I don't think this is the place to put these things, they should go to the proper migration file

Reply:
I am just making less work for people, who need to write a changelog for the next release.